### PR TITLE
fix: update ToS

### PR DESCRIPTION
**Pull request by dchoi27**

### CODE REVIEW

File: packages/website/pages/terms.js [hunk 1]
```diff
@@ -23,12 +23,14 @@ export default function TermsOfService() {
...
-          website (the “Website”) and all content, services and products
-          available at or through the Website. The Website is offered subject to
-          your acceptance without modification of all of the terms and
-          conditions contained herein. As all data uploaded to nft.storage will
+          website (the “Website”) and all content, services and products 
+          available at or through the Website (the “Service”). These terms 
+          and conditions are entered into by and between you and Protocol Labs, 
+          Inc. (“Protocol Labs”). The Website is offered subject to your 
+          acceptance without modification of all of the terms and conditions 
+          contained herein. As all data uploaded to nft.storage will
```
Comment (Codigo-Fuentes) [inline packages/website/pages/terms.js:33]: Nft.Storage ?

File: packages/website/pages/terms.js [hunk 2]
```diff
@@ -39,59 +41,58 @@ export default function TermsOfService() {
...
           <HashLink id="data-limits">Data Limits</HashLink>
         </h2>
         <p className="lh-copy">
-          The service accepts arbitrary upload sizes up to 30GB per upload.
-          Currently, the service has no data upload limit. Protocol Labs may
-          amend this term limit at its sole discretion.
+          The Service is offered for the creation and storage of NFTs. Use of the 
+          Service to store other types of data is not permitted. The Service 
+          accepts arbitrary upload sizes up to 31GiB per upload. Currently, the 
+          Service does not limit users in the total data they can upload. Protocol
```
Comment (Codigo-Fuentes) [inline packages/website/pages/terms.js:52]: Suggestion: "...does not limit users with respect to the total amount of data they can upload."

File: packages/website/pages/terms.js (hunk 1 above)
Comment (alanshaw) [reply packages/website/pages/terms.js:33]: No "NFT.Storage" ;)

File: packages/website/pages/terms.js [hunk 3]
```diff
@@ -39,59 +41,58 @@ export default function TermsOfService() {
...
         <h2 className="chicagoflf">
           <HashLink id="data-limits">Data Limits</HashLink>
         </h2>
         <p className="lh-copy">
-          The service accepts arbitrary upload sizes up to 30GB per upload.
-          Currently, the service has no data upload limit. Protocol Labs may
-          amend this term limit at its sole discretion.
+          The Service is offered for the creation and storage of NFTs. Use of the 
+          Service to store other types of data is not permitted. The Service 
+          accepts arbitrary upload sizes up to 31GiB per upload. Currently, the
```
Comment (alanshaw) [inline packages/website/pages/terms.js:51]:
```suggestion
          accepts uploads of up to 31GiB in size. Currently, the 
```

**Review by alanshaw — APPROVED** [review]
Minor tweaks but nothing blocking.

File: packages/website/pages/terms.js [hunk 4]
```diff
@@ -39,59 +41,58 @@ export default function TermsOfService() {
...
-          </a>{' '}
-          data.
+          All data uploaded via the Service is available to anyone who requests it using 
+          the correct content identifier. Users should not store any private or sensitive 
+          information in an unencrypted form using the Service. Deleting files from the 
+          Service will remove them from the file listing for a user's account, but nodes 
+          on the IPFS network may retain copies of the data indefinitely and data 
+          aggregated into Filecoin deals will persist for the duration of the relevant 
+          storage deal.
+        </p>
```
Comment (alanshaw) [inline packages/website/pages/terms.js:94]: Remove to fix tests

```suggestion
```

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/website/pages/terms.js [hunk 5]
```diff
@@ -39,59 +41,57 @@ export default function TermsOfService() {
...
-          to make the data available in IPFS. Retrieving data over the IPFS
-          network is the recommended means of accessing{' '}
-          <a className="black" href="https://nft.storage">
-            nft.storage
-          </a>{' '}
-          data.
+          All data uploaded via the Service is available to anyone who requests it using 
+          the correct content identifier. Users should not store any private or sensitive 
+          information in an unencrypted form using the Service. Deleting files from the 
+          Service will remove them from the file listing for a user's account, but nodes
```
Comment (alanshaw) [inline packages/website/pages/terms.js:90]:
```suggestion
          Service will remove them from the file listing for a user&apos;s account, but nodes 
```

**Review by alanshaw — APPROVED** [review]

**Pull request merged**